c++/src/Statistics.hh

```diff
@@ -41,49 +41,181 @@ namespace orc {
   };
 
 /**
+ * Internal Statistics Implementation
+ */
+
+  template <typename T>
+  class InternalStatisticsImpl {
+  private:
+    bool hasNull_;
```

Comment:
Not sure if this is right time to enforce a new coding style. We have an existing code base roughly about 10,000 lines. It's quite a bit of effort to convert existing code into proposed style. What's the problem with current style? Or ask it differently - what is the benefit of moving over to google style?